analyses/molecular-subtyping-HGG/02-HGG-molecular-subtyping-subset-files.R

```diff
@@ -71,14 +77,14 @@ cn_df <- read_tsv(file.path(
 
 # Read in fusion data
 fusion_df <- read_tsv(
-  file.path(data_dir, "pbta-fusion-putative-oncogenic.tsv"))
+  file.path(root_dir, "analyses","fusion_filtering", "results", "pbta-fusion-putative-oncogenic.tsv"))
```

Comment:
Is there a particular reason you didn't change the `data_dir` specification but instead changed the file path here? I noticed this happens a few times in this PR.

Reply:
since the relative analyses/fusion_filtering/results would have the updated files from https://github.com/kgaonkar6/OpenPBTA-analysis/blob/all_subtyping_gsea_rerun/scripts/run-for-subtyping.sh I pointed to thatt folder instead of data_dir. Also data_dir is used for other files as well so didn't change that variable.

Reply:
Okay. No problem. To keep it parallel, you could specify two different data_dir and call one collapse_data_dir, or whatever is appropriate, but this seems fine for the time being.